client/hotrod-client/src/main/java/org/infinispan/client/hotrod/counter/impl/RemoteCounterManager.java

```diff
@@ -60,6 +60,10 @@ public boolean defineCounter(String name, CounterConfiguration configuration) {
       return await(factory.newDefineCounterOperation(name, configuration).execute());
    }
 
+   @Override
+   public void undefineCounter(String name) {
```

Comment:
is there a JIRA to implement this?

Reply:
no

Reply:
created https://issues.jboss.org/browse/ISPN-10402 so I don't forget about it